mzLib/Test/TestMzLibUtil.cs

```diff
@@ -163,8 +169,192 @@ public void TestRemoveSpecialCharacters()
             string cleanSeq = seqWithHash.ToString();
             ClassExtensions.RemoveSpecialCharacters(ref cleanSeq, specialCharacter: "#");
             Assert.AreEqual("PEPTIDE", cleanSeq);
+        }
+
+        [Test]
+        public void TestQuantifiedModification()
+        {
+            var quantmod = new QuantifiedModification(idWithMotif: "TestMod: ModX on AAY", positionInPeptide: 1, positionInProtein: 2, intensity: 10);
```

Comment:
IdWithMotif just refers to the part after the colon. The full mod string is {Modification Type}: {Id with motif}